package.json

```diff
@@ -71,7 +71,8 @@
     "chai-immutable": "^1.5.4",
     "chalk": "^1.1.3",
     "mocha": "^3.0.2",
-    "nyc": "^8.1.0"
+    "nyc": "^8.1.0",
+    "offline-plugin": "^3.4.2"
```

Comment:
Would you be able to move `offline-plugin` from `devDependcies` -> `dependencies` in package.json. Otherwise Heroku won't build properly.